client/my-sites/site-settings/jetpack-credentials-banner/types.ts

```diff
@@ -0,0 +1,6 @@
+export type PreferenceType = 'dismiss' | 'view';
+
+export type Preference = {
+	date: number;
+	type: PreferenceType;
+};
```

Comment:
Love this! Though shouldn't this be [`types.d.ts`](https://microsoft.github.io/TypeScript-New-Handbook/chapters/type-declarations/)? Or does that not play nice with our webpack config?

Reply:
Not sure, I'll try with `types.d.ts`.

Reply:
This worked. Now the file is called `types.d.ts`.